recipe/meta.yaml

```
@@ -36,7 +36,7 @@ test:
     - conda_project
   commands:
     - conda-project --help
-    - python -c "import conda_project; ver = conda_project.__version__; assert ver == '{{ version }}'"
+    - python -c "import conda_project; ver = conda_project.__version__; assert ver == {{ version }}"
```

I would almost expect we'd need to escape instead of remove:

```suggestion
    - python -c "import conda_project; ver = conda_project.__version__; assert ver == \"{{ version }}\""
```